src/invidious/routing.cr

```diff
@@ -230,7 +232,7 @@ module Invidious::Routing
       get "/api/v1/channels/:ucid/streams", {{namespace}}::Channels, :streams
       get "/api/v1/channels/:ucid/channels", {{namespace}}::Channels, :channels
 
-      {% for route in {"videos", "latest", "playlists", "community", "search"} %}
+      {% for route in {"videos", "latest", "playlists", "community", "search", "podcasts", "releases"} %}
```

Comment:
This part is for the legacy `/api/v1/channels/<ucid>/<route>` vs; `/api/v1/channels/<route>/<ucid>` mixup.

Can you add the new routes two lines above (i.e only `/api/v1/channels/<ucid>/<route>`)

Reply:
Fixed